### PR TITLE
Add Valid HTML Types To Module info.json Files

### LINKED ISSUE #570: Map types in `info.json` files to valid HTML types
### Please describe the problem (or idea)

Input types in some module **info.json** files are not valid html input types. This makes typing number type values inconvenient and invalid values can be submitted. Types like **integer** and **string** should be replaced with **number**, **range** or **string**.

https://github.com/publiclab/image-sequencer/blob/611016b022b22d1f89247e7b13f7650e1b65f2c7/src/modules/Rotate/info.json#L6

### Possible Fixes

Add a new function in `examples/lib` which maps different types to valid HTML types

## Thank you!

### PR DESCRIPTION
**Pull request by Mridul97**
Fixes #570 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Mridul97** [comment]
For right now, I have left float type to be float, because I think it would not be a good idea to make htmltype for float to be a number or range, maybe text, would look into this and make the changes. @HarshKhandeparkar thoughts?

**Comment by HarshKhandeparkar** [comment]
Please check out https://github.com/Mridul97/image-sequencer/pull/2/files

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -213,7 +212,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
 
 
 
-    $('input[type="range"]').on('input', function() {
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js]: This will not work. It should be type="range".

**Comment by HarshKhandeparkar** [comment]
@Mridul97 I have made many other changes which are required. What should we do?

**Comment by HarshKhandeparkar** [comment]
@Mridul97 can you copy my branch and carry over the pr from there? You can do any necessary changes.

**Comment by HarshKhandeparkar** [comment]
Next year I will be in 11th grade so I will not be able to do anything. This will probably be my last contribution to image-sequencer atleast for the next 2 or 3 years.

**Comment by Mridul97** [comment]
@HarshKhandeparkar It is fine with me, if you want to open a new pr, I will close with one or I can give you access rights to this branch and you can push the changes!

**Comment by HarshKhandeparkar** [comment]
What do you want to do? You can copy my branch and open a new pr if you want or I will open one.

**Comment by Mridul97** [comment]
You can open a new one, I will close this one!

**Comment by HarshKhandeparkar** [comment]
Ok thanks a lot😊😊😊 and sorry for the inconvenience and for the time i wasted. 😣😣

**Comment by Mridul97** [comment]
No problem! :smile:

**Comment by HarshKhandeparkar** [comment]
Thanks a lot!!

**Pull request closed without merging**